src/routes/bookmark.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import bookmarkController from '../controllers/bookmarkController';
+import { verifyToken, checkBookmarkUser } from '../middlewares/authentication';
+import checkParams from '../middlewares/checkParams';
+
+const {
+  bookmark,
+  getAllBookmark,
+  unBookmark
+} = bookmarkController;
+
+const { idChecker, validateId } = checkParams;
+
+const bookmarkRouter = express.Router();
+
+bookmarkRouter.get('/users/bookmarks', verifyToken, getAllBookmark);
+bookmarkRouter.post('/articles/:slug/bookmarks', verifyToken, bookmark);
+bookmarkRouter.delete('/users/bookmarks/:id', verifyToken, idChecker, validateId, checkBookmarkUser, unBookmark);
```

Comment:
@frostyblok About your route, using the plural 'users' instead of 'user' kind of means that everyone can access that route and it's not just for one person, or what do you think?

Reply:
Hello Juwon, thank you for spotting this out. I have made all the necessary corrections. Thank you.